.github/workflows/workspace.yaml

```diff
@@ -22,15 +22,16 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: fmt
-          args: --all -- --check
+          args: --manifest-path trusted/Cargo.toml --all -- --check
```

Comment:
Why not use the working dir override?
```suggestion
          args: --all -- --check
        working-directory: ./trusted
```

Reply:
I wanted to :(, https://github.com/actions-rs/cargo/issues/6

Reply:
Fair enough. Seems worth documenting here

Reply:
added a comment